src/Stratis.Bitcoin/Consensus/CoinViews/CoinViewStack.cs

```diff
@@ -1,10 +1,10 @@
 ﻿using System.Collections.Generic;
 using Stratis.Bitcoin.Utilities;
 
-namespace Stratis.Bitcoin.Features.Consensus.CoinViews
+namespace Stratis.Bitcoin.Consensus.CoinViews
 {
     /// <summary>
-    /// Stack of coinview layers. All classes in the stack have to be based on <see cref="CoinView"/> class
+    /// Stack of coinview layers. All classes in the stack have to be based on <see cref="ICoinView"/> interface
```

Comment:
CoinViewStack can be completely removed. It's used only in tests and in ConsensusStats.
Since we have both coinviews accessible via DI we don't need to use stack in consensus stats anymore